gensim/summarization/bm25.py

```diff
@@ -264,7 +268,7 @@ def _get_scores(bm25, document):
     return bm25.get_scores(document)
 
 
-def iter_bm25_bow(corpus, k1=1.5, b=0.75, epsilon=0.25, n_jobs=1):
+def iter_bm25_bow(corpus, k1=PARAM_K1, b=PARAM_B, epsilon=EPSILON, n_jobs=1):
```

Comment:
It may be better to move k1, b, and epsilon to the end of the keyword arguments. If someone calls `get_bm25_weights(corpus, n_jobs)`, then your change would silently break their code. The same applies to function `get_bm25_weights`.